internal/server/iigointernal/orchestration.go

```diff
@@ -118,37 +118,42 @@ func RunIIGO(g *gamestate.GameState, clientMap *map[shared.ClientID]baseclient.C
 		aliveClientIds = append(aliveClientIds, shared.ClientID(int(v)))
 		resourceReports[shared.ClientID(int(v))] = iigoClients[shared.ClientID(int(v))].ResourceReport()
 	}
-	featurePresident.broadcastTaxation(resourceReports)
-	featurePresident.requestAllocationRequest()
-	featurePresident.replyAllocationRequest(g.CommonPool)
-	featurePresident.requestRuleProposal()
-	ruleToVote := featurePresident.getRuleForSpeaker()
+	executiveBranch.broadcastTaxation(resourceReports)
+	executiveBranch.requestAllocationRequest()
+	executiveBranch.replyAllocationRequest(g.CommonPool)
+	executiveBranch.requestRuleProposal()
+	ruleToVote := executiveBranch.getRuleForSpeaker()
 
 	// 3 Speaker actions
-	featureSpeaker.setRuleToVote(ruleToVote)
-	featureSpeaker.setVotingResult(iigoClients)
-	_ = featureSpeaker.announceVotingResult()
+	legislativeBranch.setRuleToVote(ruleToVote)
+	legislativeBranch.setVotingResult(aliveClientIds)
+	_ = legislativeBranch.announceVotingResult()
 
 	// 4 Declare performance (Judge) (in future all the iigointernal)
 	if judgeInspectingHistoryError != nil {
-		featureJudge.declarePresidentPerformanceWrapped()
+		judicialBranch.declarePresidentPerformanceWrapped()
 
-		featureJudge.declareSpeakerPerformanceWrapped()
+		judicialBranch.declareSpeakerPerformanceWrapped()
 	}
```

Comment:
please check errors directly when they are assigned?

Reply:
This particular process has ti be done in order, but as before I will move these error return types of booleans